README.md

```diff
@@ -1,2 +1,23 @@
 # hw01
 Repository for homework 1
+
+## Introduction
+
+#### Hello, my name is *Clara Sava-Segal*. This biography will consist of the following:
+  
+* Background
+* Course of Study and Interests
+* Plans post-graduation
+
+**Background:**
+I am currently a fourth year in the College at the University of Chicago, but I grew up right outside of [New York City](https://en.wikipedia.org/wiki/New_York_City). 
+
+**Course of Study and Interests:**
+As an undergrad, I've studied *Psychology* and *Comparative Human Development* and minored in *Biology.* I'm interested in the intersection between education and cognitive development and I worked in the [Casasanto Lab](http://www.casasanto.com), where I looked at the effects of different experiences on cognition. I am also currently working at a high school called Gary Comer College Prep.
+
+![Gary Comer College Prep Building.](https://github.com/csava1025/hw01/blob/master/GaryComer.jpg)
+
```

Comment:
Great use of the link syntax in markdown -- it works nicely and looks good in the document. Good job.